### ISSUE TITLE
Style/SpaceInsidePercentLiteralDelimiters false positive

### ISSUE THREAD

**Issue by jcoyne** [bbatsov/rubocop#4327]
I have this in ruby:
```ruby
solr_version = `curl -s 'http://lucene.apache.org/solr/' | grep -oE '\\\d+\\\.\\\d+\\\.\\\d+' | tail -1 `.chomp
```
and rubocop is reporting:
```
C: Style/SpaceInsidePercentLiteralDelimiters: Do not use spaces inside percent literal delimiters.
```
